build.gradle.kts

```diff
@@ -119,43 +117,22 @@ tasks {
     }
 }
 
-val sourcesJar by tasks.registering(Jar::class) {
-    dependsOn(JavaPlugin.CLASSES_TASK_NAME)
-    group = JavaBasePlugin.DOCUMENTATION_GROUP
-    description = "Assembles sources JAR"
-    archiveClassifier.set("sources")
-    from(sourceSets.main.get().allSource)
-}
```

Comment:
I couldn't find a difference between this, and the default Java plugin's artifact (`withSourcesJar()`). The new version of _Plugin Publishing Plugin_ automatically configures publications to include sources and our custom config caused a build error

Reply:
Nice cleanup!